keyboards/aeboards/ext65/keymaps/via/rules.mk

```
@@ -0,0 +1,68 @@
+# project specific files
+SRC =	keyboards/wilba_tech/wt_main.c
```

This should be a plus:

```suggestion
SRC +=	keyboards/wilba_tech/wt_main.c
```

it shouldn't make a difference, but ....

---

They're modified copies of the one in the parent, until we refactor them.